### ISSUE TITLE
AttributeError when connecting to OpenCV-camera

### ISSUE THREAD

**Issue by haesleinhuepf** [jacopoabramo/napari-live-recording#9]
Hi @jacopoabramo ,

I just tested napari-live-recording again and after clicking on "Connect" to my "Default Camera (OpenCV)", there is an error message popping up:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~\miniconda3\envs\nap-dev-bio2\lib\site-packages\napari_live_recording\_dock_widget.py in _on_connect_clicked(self=)
    303                 self._set_widgets_enabled(True)
    304                 self.camera_pixel_type_combobox.currentTextChanged.connect(self._on_pixel_format_changed)
--> 305                 self._set_default_roi()
        self._set_default_roi = >
    306             else:
    307                 raise CameraError(f"Error in opening {self.camera.get_name()}")

~\miniconda3\envs\nap-dev-bio2\lib\site-packages\napari_live_recording\_dock_widget.py in _set_default_roi(self=)
    428     def _set_default_roi(self):
    429         self.roi = self.camera.get_sensor_range()
--> 430         self.camera_roi_x_offset_spinbox.setValue(self.roi.offset_x)
        self.camera_roi_x_offset_spinbox.setValue = 
        self.roi.offset_x = undefined
    431         self.camera_roi_y_offset_spinbox.setValue(self.roi.offset_y)
    432         self.camera_roi_x_offset_spinbox.setSingleStep(self.roi.ofs_x_step)

AttributeError: 'NoneType' object has no attribute 'offset_x'
```

To reproduce: I just started napari, the live recoding plugin, selected the camera and clicked on "Connect".

![image](https://user-images.githubusercontent.com/12660498/147410672-cf728ee0-3862-415b-b514-96e4be6ca1cd.png)

Afterwards, when I click the start recording button, the plugin works nicely. 

Let me know if I can help tracing down that error message.

Happy holidays!

Best,
Robert

**Comment by jacopoabramo** [comment]
Hi @haesleinhuepf, I hope you had happy holidays as well. I apologize for the terrible delay, I didn't had much time to spend on this. I've attempted to fix this issue in version 0.1.7, please give it a try to see if it works out.

The ROI handling for OpenCV still is not optimal but now it should nevertheless work (after setting the ROI the first time, the subsequent changes to the ROI are applied immediately. It's not a big of a deal but that's not how it's supposed to work so I will look into that).

Also I should mention that I removed the `cv2.waitKey` function which was creating some problems as you mentioned in #5, let me know if this fixes it or it breaks things down.

**Comment by jacopoabramo** [comment]
Closing issue.
